src/components/HistoryEventDetails/layouts/EnsNameEventDetails.js

```diff
@@ -65,7 +65,15 @@ function EnsNameEventDetails({ event, chain }: Props) {
       <Text variant="large">{t('label.registered')}</Text>
       <Spacing h={spacing.extraLarge} />
 
-      {!!event?.fee && <FeeLabel value={event.fee.value} symbol={event.fee.symbol} mode="actual" chain={chain} />}
+      {!!event?.fee && (
+        <FeeLabel
+          value={event.fee.value}
+          assetSymbol={event.fee.symbol}
```

Comment:
IMO would drop assetSymbol prop, and ask FeeLabel do find it from supported assets. That would make FeeLabel easier to use.

Reply:
Per our internal discussion added this as to do: get assetSymbol from matching asset once assets can be queried by assetAddress as key instead of performing expensive search on whole assets array.